pkg/scheduler/profile/profile_test.go

```diff
@@ -273,6 +273,67 @@ func TestNewMap(t *testing.T) {
 			},
 			wantErr: "duplicate profile",
 		},
+		{
+			name: "scheduler name is needed",
+			cfgs: []config.KubeSchedulerProfile{
+				{
+					Plugins: &config.Plugins{
+						QueueSort: &config.PluginSet{
+							Enabled: []config.Plugin{
+								{Name: "QueueSort"},
+							},
+						},
+						Bind: &config.PluginSet{
+							Enabled: []config.Plugin{
+								{Name: "Bind1"},
+							},
+						},
+					},
+				},
+			},
+			wantErr: "scheduler name is needed",
+		},
+		{
+			name: "plugins required for profile",
+			cfgs: []config.KubeSchedulerProfile{
+				{
+					SchedulerName: "profile-1",
+				},
+			},
+			wantErr: "plugins required for profile",
+		},
+		{
+			name: "one queue sort plugin required for profile",
+			cfgs: []config.KubeSchedulerProfile{
+				{
+					SchedulerName: "profile-1",
+					Plugins: &config.Plugins{
+						Bind: &config.PluginSet{
+							Enabled: []config.Plugin{
+								{Name: "Bind1"},
+							},
+						},
+					},
+				},
+			},
+			wantErr: "one queue sort plugin required for profile",
+		},
+		{
+			name: "invalid framework configuration",
+			cfgs: []config.KubeSchedulerProfile{
+				{
+					SchedulerName: "invalid-profile",
+					Plugins: &config.Plugins{
+						QueueSort: &config.PluginSet{
+							Enabled: []config.Plugin{
+								{Name: "QueueSort"},
+							},
+						},
+					},
+				},
+			},
+			wantErr: "at least one bind plugin is needed",
+		},
```

Comment:
it's duplicated with https://github.com/kubernetes/kubernetes/blob/5bda0c1b3b3b3f9cbfc65255d7258fe4b441410e/pkg/scheduler/profile/profile_test.go#L71-L84.

Reply:
Thank you for your review.
I will remove it.

Reply:
Can you address this:

> Please move the cases to `TestNewProfile` as `TestNewMap` is focus on testing conflicts from multiple profiles.

Reply:
I missunderstood your comment.
However, `NewMap` does not call `validate`, so it does not occur an error in these invalid cases.
Could you tell me whether this behavior is ok or not?

Reply:
> However, NewMap does not call validate

I see, didn't notice that `NewProfile` doesn't check presence of schedulerName and plugins. Then let's just move the "one queue sort plugin required for profile" test to `NewProfile` - which is expected to throw an error.

Reply:
Thanks your review.
I will fix it.